spec/vcr_cassettes/Alert/Triggers/Templates/setup.yml

```diff
@@ -37,7 +37,7 @@ http_interactions:
       Pragma:
       - no-cache
       Date:
-      - Tue, 27 Sep 2016 18:17:14 GMT
+      - Fri, 07 Oct 2016 15:36:32 GMT
```

Comment:
I think that we should only update related cassettes to avoid cluttering the commit history.
WDYT @pilhuhn  ?